speaker_doc.py

```diff
@@ -0,0 +1,23 @@
+import json
```

Comment:
A better location for this file would be inside the `scripts` folder. Also, rename the file to something more descriptive.